config/settings/secure_example.py

```diff
@@ -1,3 +1,8 @@
+# Celery settings
+AMQP_USER = 'rabbitmq'
+AMQP_PASS = 'rabbitmq'
+SUBSCRIPTION_SCHEDULE = {'hour': '*', 'minute': '*', 'day_of_week': '*'}
```

Comment:
The default value should be meaningful, whether every minute running is ok for default value?

Reply:
done, thanks